docs/DeveloperGuide.md

```diff
@@ -1063,6 +1092,19 @@ testers are expected to do more *exploratory* testing.
        
     1. Other incorrect untag commands to try: `untag`, `untag -1 t/<valid tag name>`, `...`.<br>
        Expected: Similar to previous.
+
+### Export Mailing List Command
+1.  Export list with default attributes   
```

Comment:
Maybe shouldn;t be default attributes because the test cases got opther fields :)

Reply:
:D